src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/gnosisSafeChains.ts

```diff
@@ -0,0 +1,17 @@
+export const chains =
+  // @TODO maybe also add the chainIds for each of these for specificity
+  [
+    { value: 'arbitrum', label: 'Arbitrum' },
+    { value: 'aurora', label: 'Aurora' },
+    { value: 'avalanche', label: 'Avalanche' },
+    { value: 'binance', label: 'Binance Smart Chain' },
+    { value: 'energyweb', label: 'Energy Web Chain' },
+    { value: 'ethereum', label: 'Ethereum Mainnet' },
+    { value: 'xdai', label: 'Gnosis Chain' },
+    { value: 'goerli', label: 'Goerli' },
+    { value: 'harmony', label: 'Harmony' },
+    { value: 'optimism', label: 'Optimism' },
+    { value: 'polygon', label: 'Polygon' },
+    { value: 'rinkeby', label: 'Rinkeby' },
+    { value: 'volta', label: 'Volta' },
+  ];
```

Comment:
Two things here.

1. As I mentioned previously, the user won't necessarily change the chain themselves. 

2. These network, even if just used for display purposes, should be added to `src/modules/constants` in the same format as the other networks _(even if not all props are set for them)_

Eg:
```typescript
const XDAI_NETWORK: NetworkInfo = {
  name: 'xDAI Chain',
  chainId: 100,
  shortName: 'xDai',
  displayENSDomain: 'joincolony.colonyxdai',
  blockExplorerName: 'Blockscout',
  blockExplorerUrl: 'https://blockscout.com/poa/xdai',
  tokenExplorerLink: 'https://blockscout.com/poa/xdai/tokens',
  contractAddressLink: 'https://blockscout.com/poa/xdai/address',
  rpcUrl: 'https://rpc.xdaichain.com',
};
```